src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/runtime:6.0 AS base
```

Comment:
Nit: We usually include the hash when pulling an image to help guarantee repeatable builds. This is because an image can be rebuilt and deployed using the same tag name (i.e. runtime:latest). 

`FROM mcr.microsoft.com/dotnet/sdk:7.0.306-cbl-mariner2.0-amd64@sha256:71244bf09ad28bdcb89587bc19a5456d4ebdf5042fa0591a33903e249a8b347c AS build`

Reply:
Thank you for pointing this out. I have added the hash for the images.